.github/workflows/pull_request.yml

```
@@ -72,6 +67,18 @@ jobs:
         if: steps.diff.outputs.entries
         run: node tests/regions.js ${{ steps.diff.outputs.entries }}
 
+      - name: Validate Categories
+        if: steps.diff.outputs.entries
+        run: node tests/categories.js ${{ steps.diff.outputs.entries }}
+
+      - name: Validate Images
+        if: ${{ steps.diff.outputs.entries || steps.diff.outputs.images }}
+        run: node ./tests/images.js
```

```suggestion
        run: node tests/images.js
```

Just to keep it uniform